demo/src/app/components/dropdown/demos/basic/dropdown-basic.html

```diff
@@ -0,0 +1,8 @@
+ <div ngbDropdown [open]="true">
+    <ngbd-button class="btn btn-primary" id="dropdownMenu1" ngbDropdownToggle>Toggle dropdown</ngbd-button>
+    <div class="dropdown-menu" aria-labelledby="dropdownMenu1">
+   	    <a class="dropdown-item">Action - 1</a>
```

Comment:
Indentation seems off (first item is indented differently as compared to others)